commons-hpcc/src/main/java/org/hpccsystems/commons/ecl/FieldDef.java

```diff
@@ -35,6 +35,7 @@
     private long              len                = 0;
     private boolean           fixedLength        = false;
     private boolean           isUnsigned         = false;
+    private int               additionalFlags    = 0;
```

Comment:
is there a known set of valid additional flags?

Reply:
At the moment there is only one flag that we need to give to dafilesrv, but we do not use ourselves. However, the platform code is structured in such a way that there could potentially be more flags that fall into this "track only" category. All of these flags would be 0x8000 or above.